tests/functional/tests.py

```diff
@@ -15,9 +15,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import sys
-sys.path.append('/home/vadmeste/work/python/minio-py')
```

Comment:
> oh wow when was this added?

Few weeks ago, it was when I restructured tests, which looks like a big change in diff.